isamples_metadata/Transformer.py

```diff
@@ -54,9 +52,24 @@ def transform(self) -> typing.Dict:
             },
             "registrant": self.sample_registrant(),
             "samplingPurpose": self.sample_sampling_purpose(),
+            "curation": {
+                "label": self.curation_label(),
+                "description": self.curation_description(),
+                "accessConstraints": self.curation_access_constraints(),
+                "curationLocation": self.curation_location(),
+                "responsibility": self.curation_responsibility(),
+            },
+            "relatedResource": self.related_resources()
```

Comment:
These are all new fields we'll need to ingest in solr.  They aren't present for SESAR.

Reply:
@datadavev please confirm

Reply:
Agreed. Some of this information may become available from SESAR as well, though isn't in the records yet.